src/NmeaParser.Shared/Nmea/Gps/GPRMC.cs

```diff
@@ -39,7 +39,7 @@ protected override void OnLoadMessage(string[] message)
 			if (message == null || message.Length < 11)
 				throw new ArgumentException("Invalid GPRMC", "message"); 
 			
-			if (message[8].Length == 6 && message[0].Length == 6)
+			if (message[8].Length == 6 && message[0].Length >= 6)
 			{
```

Comment:
Why would the length ever be greater than 6? That doesn't seem to be part of the spec?